css/common/reset.css

```
@@ -0,0 +1,37 @@
+/* 기본 스타일 초기화 */
+
+/* margin, padding 초기화 */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* 글꼴 설정 (모든 요소에 대해) */
+body, h1, h2, h3, h4, h5, h6, p, a, li, ul, ol {
+    font-family: "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont, system-ui, Roboto, "Helvetica Neue", "Segoe UI", "Apple SD Gothic Neo", "Noto Sans KR", "Malgun Gothic", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
+}
```

💊 제안
font-family는 상속되는 속성이라 body 태그에 명시해주시고 나머지는 inherit로 받으시는 것을 추천드려요~

```suggestion
body {
   font-family: "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont, system-ui, Roboto, "Helvetica Neue", "Segoe UI", "Apple SD Gothic Neo", "Noto Sans KR", "Malgun Gothic", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
}

h1, h2, h3, h4, h5, h6, p, a, li, ul, ol {
   font-family: inherit;
}
```